### ISSUE TITLE
Bug: importing CSS props with postcss will throw error for: missed semicolon 

### ISSUE THREAD

**Issue by GrimLink** [fylgja/fylgja#191]
This is caused by the export rules, for example with `@fylgja/sizes` package.json there is an explicit export rule for specific files:

```json
{
  "name": "@fylgja/sizes",
  ...
  "type": "module",
  "main": "index.cjs",
  "unpkg": "sizes.css",
  "style": "sizes.css",
  "module": "index.js",
  "exports": {
    ".": {
      "import": "./index.js",
      "require": "./index.cjs",
      "default": "./index.cjs"
    }
  },
  ...
}
```

Sadly the `style` export rule is not used when combined with the `exports` export rule.

To resolve this, the css export needs used as the default export option in `exports` export rule until postcss adds support for this, see: https://github.com/postcss/postcss-import/issues/492
or Fylgja needs a second export option for CSS

**Comment by GrimLink** [comment]
After conducting tests within the https://github.com/fylgja/examples environment, we successfully resolved the issue by setting the default preference to the CSS file. Given that Fylgja prioritizes CSS as its primary focus, we intend to establish a default rule for scenarios where both CSS and JS coexist in the same location, giving precedence to CSS exports.

Furthermore, we will include explicit exports for both cases as follows:

```json
"exports": {
  "./css": "./sizes.css",
  "./js": {
    "import": "./index.js",
    "require": "./index.cjs",
    "default": "./index.cjs"
  }
}
```

**Comment by GrimLink** [comment]
Fixed in [v1.3.4](https://github.com/fylgja/fylgja/releases/tag/1.3.4)

This had been tested with node 16 and vite v4.2 using Fylgja v1.3.4 in our examples repo [Fylgja Examples - Vite PostCSS](https://github.com/fylgja/examples/tree/main/vite-postcss).
The `style` export option in the `exports` of the `package.json` is used when added before the `import` and `require`.

For an example see the [Fylgja Sizes component package.json](https://github.com/fylgja/fylgja/blob/main/components/sizes/package.json#L13-L27)
